call-home/src/bin/stats/store/events_store.rs

```diff
@@ -0,0 +1,132 @@
+use crate::cache::events_cache::{Cache, EventSet};
+use k8s_openapi::api::core::v1::ConfigMap;
+use kube::{
+    api::{Api, Patch, PatchParams, PostParams},
+    core::ObjectMeta,
+    Client,
+};
+use obs::common::{
+    constants::{EVENT_STATS_DATA, EVENT_STORE, EVENT_STORE_LABLE_KEY, PATCH_PARAM_FILED_MANAGER},
+    errors,
+    utils::release_name,
+};
+use snafu::ResultExt;
+use std::{collections::BTreeMap, ops::DerefMut, time::Duration};
+use tracing::info;
+
+/// Initialize a config map for storing events.
+pub async fn initialize(namespace: &str) -> errors::Result<ConfigMap> {
```

Comment:
Doesn't need to be pub

Reply:
merging this PR . will address this in another PR.